pages/common/more.md

```diff
@@ -1,12 +1,21 @@
 # more
 
-> Open a file for interactive reading, allowing scrolling and search (in forward direction only).
+> Open a file for interactive reading, allowing scrolling and search.
+> More information: <https://manned.org/more>.
 
 - Open a file:
 
-`more {{source_file}}`
+`more {{path/to/file}}`
 
-- Page down:
+- Open a file displaying from a specific line:
```

Comment:
```suggestion
- Open a file at a specific line:
```
Maybe that will read better?

Reply:
Because, as a user, you see the full page not just a specific line.
The `--help` says
```
display file beginning from line number
```
The `man more` says
```
+number
    Start displaying each file at line number.
```

Reply:
This, perhaps?
```suggestion
- Open a file, starting at a specific line:
```

Reply:
`beginning from` for me suggests that it does not show anything before that line - only that line and after, as does `starting at`.

`Open a file at a specific line` suggests to me it opens the entire file and jumps to a preset line to start with, but you can browse the entire file as you like.
`